corelibs/README.md

```diff
@@ -12,9 +12,10 @@ Read more about the reference project repository [here](../project.md).
 There are following Phoenix-RTOS libraries:
 
 - [Graphics library](libgraph.md) called `libgraph`, which provides drivers for graphic adapters,
-- Common Gateway Interface library called `libcgi`,
-- Video Graphics Array access library called `libvga`, used in graphic adapters' implementation, for example Cirrus,
-- Virtual I/O Device library called `libvirtio`, used for device emulation,
+- [Common Gateway Interface library](libcgi.md) called `libcgi`,
+- [Video Graphics Array access library](libvga.md) called `libvga`, used in graphic adapters' implementation,
+  for example Cirrus,
+- [Virtual I/O Device library](libvirtio.md) called `libvirtio`, used for device emulation,
 - [Universally Unique identifiers' library](libuuid.md) called `libuuid`,
 - [Cache library](libcache.md) called `libcache` which provides the user with n-way set-associative cache,
```

Comment:
Let's remove those in the re`===` in libuuid and libswdg.

![Screenshot from 2024-03-13 14-21-27](https://github.com/phoenix-rtos/phoenix-rtos-doc/assets/77304431/561c7a98-3d0a-4ff9-b83c-11f793a003b9)

![Screenshot from 2024-03-13 14-21-35](https://github.com/phoenix-rtos/phoenix-rtos-doc/assets/77304431/cd07138a-c085-4492-b0a7-0a8f4048c392)